internal/endorser/endorser_test.go

```diff
@@ -175,6 +174,34 @@ func TestLoadAndVerify_InconsistentNotVerified(t *testing.T) {
 	}
 }
 
+func TestLoadAndVerifyProvenances_NotVerified(t *testing.T) {
+	tempPath1, err := copyToTemp("../../testdata/amber_provenance.json")
+	if err != nil {
+		t.Fatalf("Could not load provenance: %v", err)
+	}
+
+	referenceValues, err := common.LoadReferenceValuesFromFile("../../testdata/different_reference_values.toml")
+	if err != nil {
+		t.Fatalf("Could not load reference values: %v", err)
+	}
+
+	_, err = loadAndVerifyProvenances(referenceValues, []string{"file://" + tempPath1})
```

Comment:
This can be inlined I think:

```
if _, err = loadAndVerifyProvenances(referenceValues, []string{"file://" + tempPath1}); err != nil { ... }
```

Reply:
Actually, we can't, I need `err` for the subsequent checks.